app/config/variables.php

```diff
@@ -288,6 +288,24 @@
                 'question' => '',
                 'filter' => 'password'
             ],
+            [
+                'name' => '_APP_PROJECT_DB',
+                'description' => 'A list of comma separated key value pairs for Project DBs where key is the database name and value is the DSN connection string.',
```

Comment:
We should probably have an example here. We might want to skip this from the docs for now as we discussed.

Reply:
Added an example 👍